helper-cli/src/main/kotlin/commands/repoconfig/GenerateScopeExcludesCommand.kt

```diff
@@ -115,6 +115,13 @@ private fun getScopeExcludesForPackageManager(packageManagerName: String): List<
                 comment = "Packages for development only."
             )
         )
+        "Conan" -> listOf(
+            ScopeExclude(
+                pattern = "build_requires",
+                reason = ScopeExcludeReason.DEV_DEPENDENCY_OF,
```

Comment:
Shouldn't this be `ScopeExcludeReason.BUILD_DEPENDENCY_OF` (also regading the comment then)?

Reply:
I decided for dev dependencies based on https://docs.conan.io/en/1.35/devtools/build_requires.html.
Have you read this?

Reply:
Yes, I've read this, but I got confused with our own "DEV" vs "BUILD" definition. I agree to the current state.